content/projects/morse-code/_index.md

```diff
@@ -5,7 +5,7 @@ flavours:
 - any_language
 prerequisites:
   hard:
-  - topics/intro-to-asserive-programming
+  - topics/intro-to-assertive-programming
```

Comment:
This is now a broken link.

If I ask you to open a file named "intro-to-assertive-programming" and yet the file doesn't exist then you wont be able to open the file.
If you update a link anywhere then you need to make sure it points to something that exists
If you change a file path then you need to make sure that everything that refers to that file is updated with the new path

This stuff is connected.